python/arcticdb/util/storage_test.py

```diff
@@ -0,0 +1,241 @@
+    )
+
+
+def verify_library(ac):
+    libraries = get_test_libraries(ac)
+    for lib_name in libraries:
+        lib = ac[lib_name]
+
+        symbols = lib.list_symbols()
+        assert len(symbols) == 3
+        for sym in ["one", "two", "three"]:
+            assert sym in symbols
+        for sym in symbols:
+            df = lib.read(sym).data
+            column_names = df.columns.values.tolist()
+            assert column_names == ["x", "y", "z"]
+
+
+def is_strategy_branch_valid_format(input_string):
+    pattern = r"^(linux|windows)_cp3(6|7|8|9|10|11).*$"
+    match = re.match(pattern, input_string)
+    return bool(match)
+
+
+def seed_library(ac, version: str = ""):
+    strategy_branch = os.getenv("ARCTICDB_PERSISTENT_STORAGE_STRATEGY_BRANCH")
+
+    if not is_strategy_branch_valid_format(strategy_branch):
+        raise ValueError(f"The strategy_branch: {strategy_branch} is not formatted correctly")
+
+    lib_name = f"seed_{version}{strategy_branch}"
+    lib_name = normalize_lib_name(lib_name)
+
+    # Each branch should create its own seed and it should be fresh on each run
+    # so delete the library, if it exists / wasn't cleaned on a previous run
+    ac.delete_library(lib_name)
+    ac.create_library(lib_name)
+
+    library = ac[lib_name]
+
+    one_df = test_df_3_cols()
+    library.write("one", one_df)
+
+    two_df = test_df_3_cols(1)
+    library.write("two", two_df)
+
+    two_df = test_df_3_cols(2)
+    library.append("two", two_df)
+
+    three_df = test_df_3_cols(3)
+    library.append("three", three_df)
+
+
+def cleanup_libraries(ac):
+    for lib in get_seed_libraries(ac):
+        ac.delete_library(lib)
+
+    for lib in get_test_libraries(ac):
+        ac.delete_library(lib)
+
+
+def check_last_timestamp(df, end_timestamp="1/1/2023"):
+    # Convert end_timestamp to datetime object if it's a string
+    if isinstance(end_timestamp, str):
+        end_timestamp = datetime.strptime(end_timestamp, "%m/%d/%Y")
+    # Check if the last timestamp in df is equal to end_timestamp
+    is_correct = df["timestamp"].iloc[-1] == end_timestamp
+    return is_correct
+
+
+def generate_pseudo_random_dataframe(n, freq="S", end_timestamp="1/1/2023"):
+    """
+    Generates a Data Frame with 2 columns (timestamp and value) and N rows
+    - timestamp contains timestamps with a given frequency that end at end_timestamp
+    - value contains random floats that sum up to approximately N, for easier testing/verifying
+    """
+    # Generate random values such that their sum is equal to N
+    values = np.random.dirichlet(np.ones(n), size=1) * n
+    # Generate timestamps
+    timestamps = pd.date_range(end=end_timestamp, periods=n, freq="S")
+    # Create dataframe
+    df = pd.DataFrame({"timestamp": timestamps, "value": values[0]})
+    return df
+
+
+def gen_fake_ticker(val):
+    # We are adding + 1 to skip the 0(empty) char
+    return f"AST_{chr((val % 255) + 1)}"
```

Comment:
`AST` is a Man Group thing, let's use dummy ISIN codes https://en.wikipedia.org/wiki/International_Securities_Identification_Number

Reply:
I wanted to have just some string that changes deterministically, so I would prefer something like TK_{..} or something.
Is that okay?